SPECS/cal10n/cal10n.spec

```
@@ -16,16 +15,16 @@
 #
 Summary:        Compiler assisted localization library (CAL10N)
 Name:           cal10n
-Version:        0.7.7
-Release:        6%{?dist}
+Version:        0.8.1.10
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Development/Libraries/Java
 URL:            http://cal10n.qos.ch
-Source0:        https://github.com/qos-ch/cal10n/archive/refs/tags/v_%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        build.xml-0.7.7.tar.xz
-Patch0:         cal10n-0.7.7-sourcetarget.patch
+Source0:        https://code.opensuse.org/adrianSuSE/%{name}/blob/factory/f/%{name}-%{version}.tar.xz
+# cal10n-build.tar.gz imported from https://code.opensuse.org/adrianSuSE/cal10n/blob/factory/f/cal10n-build.tar.xz
+Source1:        %{name}-build.tar.xz
```

If it's coming from an upstream URL, then you can use the same format as for `Source0`. Having said that, can you add a comment explaining what this source is used for, so it's easier for the next person to understand how this package is built and what they may need to look into during a future update?
```suggestion
Source1:        https://code.opensuse.org/adrianSuSE/cal10n/blob/factory/f/%{name}-build.tar.xz
```

---

Got it. Updated source1 and added comment.